src/client/register.rs

```diff
@@ -124,10 +108,10 @@ impl fmt::Debug for RegisterWrite {
         write!(
             formatter,
             "RegisterWrite::{}",
-            match *self {
-                RegisterWrite::New(_) => "NewRegister",
-                RegisterWrite::Delete(_) => "DeleteRegister",
-                RegisterWrite::Edit(_) => "EditRegister",
+            match self {
+                RegisterWrite::New(register) => format!("New({:?})", register.address()),
```

Comment:
Perhaps we want `NewRegister({:?})` not only to keep consistency with `DeleteRegister({:?})` but also to see what's new/edited?

Reply:
Delete should be just "Delete" I can update that. 

The problem with writing the whole register is that the log will become too verbose. In the context of the nodes we need not know what the content of the new register is right? That can be in the layer where data operations are performed?

Reply:
ok, yeah i guess it depends who is logging this, i just thought that if you blindly log it you don't know if it's a New Register or new what...

Reply:
Yeah. In this context it will always be `RegisterWrite::New(register_address)`

Reply:
ok, we can change it if it's not good enough in the future. But that's what I mean, you won't get `RegisterWrite::New(register_address)` written to the log, but just `New(register_address)`

Reply:
`RegisterWrite` is mentioned in L110

Reply:
ahhhh!!! I was missing that completely! sorry about that!